explorer-ui/_types/global.d.ts

```diff
@@ -5,6 +5,7 @@ declare global {
   interface Window {
     __RUNTIME_CONFIG__: {
       REACT_APP_SQUID_ENDPOINT: string;
+      REACT_APP_SQUID_ARCHIVE_ENDPOINT: string;
```

Comment:
Is this going to be used in multiple instances? Maybe it's nice to have them defined  in `.env` to prevent changing too.

Reply:
we can fix this in a separate pr?

Reply:
btw, we already use `.env` files after deploying